vpr/src/draw/intra_logic_block.cpp

```diff
@@ -380,7 +380,7 @@ static void draw_internal_pb(const ClusterBlockId clb_index, t_pb* pb, const ezg
             int type_len = strlen(pb_type->name);
             int name_len = strlen(pb->name);
             int tot_len = type_len + name_len;
-            char* blk_tag = (char*)vtr::malloc((tot_len + 8) * sizeof(char));
+            char* blk_tag = new char[(tot_len + 8)];
```

Comment:
I don't like the "8" magic number.  Same comment as above: make a C++ string, or make a defined constant (and be sure 8 is always enough, or increase it).